src/controllers/weightDriftAuditController.ts

```
@@ -0,0 +1,291 @@
+ *         description: List of audits with pagination
+ */
+export const listWeightDriftAudits = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): Promise<void> => {
+  try {
+    const { status, limit = 20, offset = 0 } = req.query;
+
+    const result = await weightDriftAuditService.listAudits(
+      (status as "pending" | "approved" | "rejected" | undefined),
```

ESLint

[error] Replace `(status·as·"pending"·|·"approved"·|·"rejected"·|·undefined)` with `status·as·"pending"·|·"approved"·|·"rejected"·|·undefined`

(prettier/prettier)